addon/components/cookie-banner/style.scss

```diff
@@ -0,0 +1,20 @@
+#cookie-banner {
+    width: 100%;
+    position: fixed;
+    bottom: 0;
+    background-color: #263947;
```

Comment:
Also, link color on background fails contrast ratio checks:
<img width="715" alt="screen shot 2018-07-10 at 4 53 34 pm" src="https://user-images.githubusercontent.com/1322421/42537133-0b0d388e-8462-11e8-9c76-4eb892171bd5.png">

Reply:
@caseyrollins @laurenbarker Unfortunately they'll need to change though, see above comment

Reply:
@caseyrollins @alexschiller what do you think of underlined white text for the links instead of the blue link color?

Reply:
Works for me!

Reply:
@erinspace, can you update your PR accordingly? Thank you!